### PR TITLE
ci(components): setup utils commitizen, husky, commitlint

### PR DESCRIPTION
**Pull request by softwarrior**
setup utils commitizen, husky, commitlint

META DATA: issue5

usage: npx git-cz

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -6,19 +6,37 @@
   "scripts": {
     "dev": "vite",
     "build": "tsc && vite build",
-    "preview": "vite preview"
+    "preview": "vite preview",
+    "prepare": "husky install"
   },
   "dependencies": {
+    "commitizen": "^4.2.5",
```
Comment (dpolevodin) [inline package.json:13]: Maybe commitizen should also be moved to "devDependencies"?

**Review by dpolevodin — APPROVED** [review]

File: config/git/commitizen.cjs [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+      name: "perf:      Изменения направленные на улучшение производительности",
+    },
+    {
+      value: "refactor",
+      name: "refactor:  Правки кода без исправления ошибок или добавления новых функций",
+    },
+    { value: "revert", name: "revert:    Откат на предыдущие коммиты" },
+    {
+      value: "style",
+      name: "style:     Правки по кодстайлу (табы, отступы, точки, запятые и т.д.)",
```
Comment (makhnatkin) [inline config/git/commitizen.cjs]: use english plz

File: config/git/commitizen.cjs [hunk 3]
```diff
@@ -0,0 +1,74 @@
+"use strict"
+
+module.exports = {
+  // Добавим описание на русском языке ко всем типам
```
Comment (makhnatkin) [inline config/git/commitizen.cjs]: use english plz

File: package.json (hunk 1 above)
Comment (softwarrior) [reply package.json:13]: Thank you!  added to devDependencies

File: config/git/commitizen.cjs (hunk 2 above)
Comment (softwarrior) [reply config/git/commitizen.cjs]: Thank you! translated into English

File: config/git/commitizen.cjs (hunk 3 above)
Comment (softwarrior) [reply config/git/commitizen.cjs]: Thank you! translated into English

**Comment by softwarrior** [comment]
All remark have been corrected. I will be grateful for the approve!

**Review by Hellomyguest — APPROVED** [review]

**Pull request merged**